dags/dbt_advanced_utility.py

```diff
@@ -0,0 +1,47 @@
+from airflow import DAG
+from airflow.operators.dummy import DummyOperator
+from airflow.operators.bash import BashOperator
+from airflow.utils.dates import datetime
+from airflow.utils.task_group import TaskGroup
+from include.dbt_dag_parser import DbtDagParser
+
+DBT_PROJECT_DIR = '/usr/local/airflow/dbt'
+DBT_GLOBAL_CLI_FLAGS = '--no-write-json'
+DBT_TARGET = 'dev'
+DBT_TAG = 'tag_staging'
+
+default_args = {
+    'owner': 'astronomer',
+    'depends_on_past': False,
+    'start_date': datetime(2020, 12, 23),
+    'email': ['noreply@astronomer.io'],
+    'email_on_failure': False
+}
+
+dag = DAG(
+    'dbt_advanced_dag_utility',
+    default_args=default_args,
+    description='A dbt wrapper for Airflow using a utility class to map the dbt DAG to Airflow tasks',
+    schedule_interval=None,
+)
+
+with dag:
+
+    start_dummy = DummyOperator(task_id='start')
+    # We're using the dbt seed command here to populate the database for the purpose of this demo
+    dbt_seed = BashOperator(
+        task_id='dbt_seed',
+        bash_command=f'dbt {DBT_GLOBAL_CLI_FLAGS} seed --profiles-dir {DBT_PROJECT_DIR} --project-dir {DBT_PROJECT_DIR}'
+    )
+    end_dummy = DummyOperator(task_id='end')
+
+    dag_parser = DbtDagParser(dag=dag,
```

Comment:
Would you mind adding a quick comment here about what this parser does?  I think it would be helpful for users looking at this for the first time.  I have a feeling some may think it's an operator of sorts when it really helps dynamically generate tasks for this DAG.